cli/version.rs

```diff
@@ -1,9 +1,15 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-pub const DENO: &str = env!("CARGO_PKG_VERSION");
 pub const GIT_COMMIT_HASH: &str = env!("GIT_COMMIT_HASH");
 pub const TYPESCRIPT: &str = crate::js::TS_VERSION;
 
+pub fn deno() -> String {
+  let semver = env!("CARGO_PKG_VERSION");
+  option_env!("DENO_CANARY").map_or(semver.to_string(), |_| {
+    format!("{}-canary.{}", semver, GIT_COMMIT_HASH)
```

Comment:
the "$VERSION-canary.$HASH" seems overly specific. Why not just "$VERSION-$HASH" ?

Reply:
Also I guess we can remove the git hash from the `--version` output too since it would be repeated twice.

Reply:
> Why not just "$VERSION-$HASH" ?

sure

> Also I guess we can remove the git hash from the `--version` output too since it would be repeated twice.

I dont mind, tho then it wont be shown in release versions; is that alright?

Reply:
> I dont mind, tho then it wont be shown in release versions; is that alright?

fine by me